lessons/Data-Science/AI/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,335 @@
+
+@lesson-instruction{
+- We will complete @printable-exercise{level-1.adoc} together, starting with "age" as the root node.
+- As we create the first level of our tree, you will discover a tool that we use to complete all the levels of our model: the decision stump.
+}
+
+@teacher{
+A @vocab{decision stump} is a one-level decision tree that makes a prediction based on the value of just a single input feature.
+
+Students are introduced to the @vocab{decision stump} in question 4 on @printable-exercise{level-1.adoc}. Be prepared to guide students through building the decision stump. They will need to use (1) the rule that you wrote in question 1 and (2) the training dataset.
+
+- The stump has three branches because we are considering customers in their teens, twenties and thirties.
+- The left-most leaf node ("teens") represents the five teens in our training dataset: Jan (16), Jose (19), Jillian (14), Ariella (16), and Danial (19).
+** Jan, Jose, and Jillian did *not* purchase the game, so they are represented by the letter N (for "no").
+** Ariella and Danial *did* purchase the game, so they are represented by the letter Y (for "yes").
+** We illustrate the teens' decisions with the following shorthand: N N N Y Y
+- Finally, we place checkmarks below correctly predicted values to indicate how successful our rule was.
+** Our rule predicted that individuals in their teens would *not* purchase the game, so we place checkmarks by the Ns that represent Jan, Jose, and Jillian. Our rule was correct.
+** We leave the Ys *without* checkmarks; our rule was wrong for Danial and Ariella.
+- We complete the same process for customers in their twenties and in their thirties, representing each individual in the training dataset with a "Y" or "N", and then placing checkmarks to indicate when our rule was successful.
+}
+
+As we move down the tree, our job is to figure out _which questions_ we should ask and _when_ we should ask them... just like when we play 20 Questions! Decision stumps will help us decide which questions produce a greater information gain.
+
+@strategy{Why Start the Tree with "Age"?}{
+Students will likely notice that we seemingly arbitrarily started the tree with "age" as the root node. _Extremely perceptive_ students may notice that for both "age" and "interest", the likelihood of a correct prediction is 10/14. In other words, starting with "interest in game" produces the same information gain as starting with "age" as the root... *so how do we decide?*
+
+It turns out there's more than one correct way to build a decision tree! In general, however, we want to avoid tall, skinny trees that pose one useless question after the other. Rather, it is beneficial to start with an attribute that will result in a _wider_ tree.
+
+Because the "age" node splits _three_ ways and the "interest in game" node splits _two_ ways, we opt to start the tree with "age".
```

Comment:
This sentence emphasizes the importance for me of introducing the diagramming of a 3-way split in the lesson plan before you send them off to work on `level-1`. I might even include two images right here to illustrate the difference you are discussing.